pkg/storageincentives/agent.go

```diff
@@ -298,8 +300,23 @@ func (a *Agent) claim(ctx context.Context) error {
 	}
 
 	if isWinner {
+		a.nodeState.SetPhase(winner)
```

Comment:
againt, winner and frozen are not "phases" they are more like the current state,
claim, reveal, and commit are the real phases

Reply:
Alright. So going to add the state separately.